orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -1123,37 +1123,6 @@ def run_step(model, input):
         _test_helpers.assert_values_are_close(ort_input.grad, pt_input.grad)
 
 
-@pytest.mark.parametrize("operator", ["min", "max"])
```

Comment:
I think you can use some feature of pytest  to skip this test currently with comments.

Reply:
Thanks for the reminder! To disable for now is better.